certified-connectors/cioplenu/apiDefinition.swagger.json

```diff
@@ -0,0 +1,114 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "cioplenu",
+    "description": "Create tasks on your cioplenu instance.",
```

Comment:
Could this description be expanded a bit? It'd be great to provide a high level overview of the cioplenu service, as well as the mention of functionality the connector offers.

Reply:
Sure. I updated the description with a more general sentence about cioplenu and what this connector can do.

Reply:
Thanks!